### PR TITLE
Script support for cambrinth_invoke_exact_amount

### PR DESCRIPTION
**Pull request by jandersson**
Updates scripts to make use of DRCA update for setting `cambrinth_invoke_exact_amount`

### CODE REVIEW

File: combat-trainer.lic [hunk 1]
```diff
@@ -2908,7 +2913,11 @@ class GameState
                 else
                   charges
                 end
-    @charges = cambrinth.nil? || cambrinth.empty? ? nil : cambrinth.dup
+    @charges = nil
```
Comment (rpherbig) [inline combat-trainer.lic]: Should `@charges_total` also be set to `nil` here?

**Comment by rpherbig** [comment]
Looks good to me. How long do you want to let this sit before merging?

**Comment by jandersson** [comment]
Id give it another day. I can set a reminder in slack

**Comment by jandersson** [comment]
Give it another day

On April 20, 2018 18:43:11 Robert Herbig <notifications@github.com> wrote:
Looks good to me. How long do you want to let this sit before merging?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


Sent with AquaMail for Android
https://www.mobisystems.com/aqua-mail

**Pull request merged**